atom/app/atom_content_client.cc

```diff
@@ -204,4 +204,18 @@ void AtomContentClient::AddServiceWorkerSchemes(
   service_worker_schemes->insert(url::kFileScheme);
 }
 
+void AtomContentClient::AddSecureSchemesAndOrigins(
+    std::set<std::string>* secure_schemes,
+    std::set<GURL>* secure_origins) {
+  std::vector<std::string> schemes;
+  ConvertStringWithSeparatorToVector(&schemes, ",",
+                                     switches::kSecureSchemes);                        
+  if (!schemes.empty()) {
```

Comment:
I think this check would be okay to remove since the for loop should handle an empty schemes vector fine.